src/components/gallery/GalleryHeader.tsx

```diff
@@ -0,0 +1,9 @@
+const GalleryHeader = () => {
+  return (
+    <div className="font-banco flex w-full flex-row justify-center text-left text-[108px] font-medium leading-[103.25px]">
```

Comment:
What does leading- do ?

 I dont think its needed

Reply:
Ah it sets the line height of the text, I was trying to copy the line height typography properties from the Figma design, but yea you're right its definitely not needed in this. I'll remove it